### DISCUSSION TITLE
HMI icon button overlay

### DISCUSSION THREAD

**Discussion by adamevess** [jilleb/MQB-FPA#56] [Q&A]
Hi, Sir

Someone know that why have abnormal status for icon overlay

![image](https://user-images.githubusercontent.com/82160973/155864706-82b958ab-9445-4ddc-97cc-2afb221ca83e.png)
I am kodiaq MY18, dataset is M1,
but when I write FPA dataset using MB format, but HMI button icon is overlay
it maybe wrong FEC format cause?  or where can control these icons

thanks!

**Comment by olli991** [comment]
Do you have both datasets available? I would like to investigate if i can see some changes that could produce this.

**Reply by adamevess** [reply]
Your available is mean normally status for icon layer? This question confused me so long, my friends of kodiaq 19 using is normal, I think maybe FPA position is important key?
